docs/content/docs/plugins/organization.mdx

```diff
@@ -1156,6 +1156,11 @@ Table Name: `invitation`
       type: "string",
       description: "The status of the invitation"
     },
+    {
+      name: "createdAt",
```

Comment:
It a duplicate here. This is already exists below.

Reply:
Gotcha. Interesting that it's here in the docs and not in the code :D